service/src/main/kotlin/no/nav/su/se/bakover/service/ProdServiceBuilder.kt

```diff
@@ -78,9 +74,6 @@ object ProdServiceBuilder : ServiceBuilder {
             brevService = brevService
         )
         val opprettVedtakssnapshotService = OpprettVedtakssnapshotService(databaseRepos.vedtakssnapshot)
-        val journalførIverksettingService = JournalførIverksettingService(databaseRepos.behandling, brevService)
-        val distribuerIverksettingsbrevService =
-            DistribuerIverksettingsbrevService(brevService, databaseRepos.behandling)
         val ferdigstillIverksettingService = FerdigstillSøknadsbehandingIverksettingServiceImpl(
```

Comment:
Mistenker at det mangler en statistikk observatør her.

Reply:
Dette gjelder også for iverksettingsservicen.

Reply:
done